src/components/featured-article/_macro-options.md

```diff
@@ -0,0 +1,37 @@
+| Name         | Type                                 | Required              | Description                                                                                                                                                 |
+| ------------ | ------------------------------------ | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| title        | `Object<Title>`                      | true                  | An object containing the article's title text and link. See [Title](#title).                                                                                |
+| metadata     | `Object<Metadata>`                   | false                 | An object containing metadata such as article type and release date. See [Metadata](#metadata).                                                             |
+| chart        | `Chart` [_(ref)_](/components/chart) | true if image not set | Configuration object for the chart.                                                                                                                         |
+| image        | `Image` [_(ref)_](/components/image) | true if chart not set | Configuration object for the image.                                                                                                                         |
+| headingLevel | number                               | false                 | Number used to determine the heading level of the title. Use to ensure the title has a correct semantic order on the page, defaulting to 2 if not provided. |
+| description  | string                               | false                 | A short HTML extract of text (for example, a short sentence to give some context of the article)                                                            |
+
+## Title
+
+| Property | Type   | Required | Description                          |
+| -------- | ------ | -------- | ------------------------------------ |
+| text     | string | true     | The article title text.              |
+| url      | string | true     | The URL linked to the article title. |
```

Comment:
Is there the potential that you would not have a url?

Reply:
No, It will always have a url